internal/graph/check.go

```diff
@@ -57,7 +57,6 @@ func (r *ResolveCheckResponse) GetResolutionMetadata() *ResolutionMetadata {
 	if r != nil {
 		return r.ResolutionMetadata
 	}
-
```

Comment:
Unrelated code change. To keep this PR history clean, would you mind reverting this change please?